src/app/home/home.component.ts

```diff
@@ -54,11 +54,14 @@ export class HomeComponent implements AfterViewInit, OnDestroy {
   private toolToSlug = {};
   private vizabiInstances = {};
   private vizabiModelChangesSubscription: Subscription;
+  private vizabiChartTypeChangesSubscription: Subscription;
   private vizabiCreationSubscription: Subscription;
   private initialToolsSetupSubscription: Subscription;
   private localeChangesSubscription: Subscription;
   private toolChangesSubscription: Subscription;
   private routesModelChangesSubscription: Subscription;
+  private initialVizabiModelIndicators = {axis_x: {}, axis_y: {}, size: {}, select: []};
```

Comment:
@vdmchpg color, labels

Reply:
@vdmchpg it's not initial, I guess..
create const INITIAL_VIZABI_MODEL_INDICATORS = {axis_x: {}, axis_y: {}, size: {}, select: []};
and then apply everywhere you need